Client/src/components/contanier/staff/Student/EditStudent/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react'
+import LabeledInput from './../../../../common/LabeledInput'
+import Button from './../../../../common/Button';
+import { studentSchema } from './../../../../../helpers/validation-schema'
+
+import './index.css';
+
+const leftSectionInfo = [
+  ['firstname', 'First name'],
+  ['lastname', 'Last name'],
+  ['level', 'Level'],
+  ['phonenumber', 'Phone number '],
+];
+const rightSectionInfo = [
+  ['mobilenumber', 'Mobile number'],
+  ['address', 'Address'],
+  ['password', 'Password']
+];
+
```

Comment:
move the leftSectionInfo and rightSectionInfo to static data external file
and then import them